ext/ffi/lib.rs

```diff
@@ -1059,14 +1066,37 @@ fn ffi_parse_pointer_arg(
   arg: v8::Local<v8::Value>,
 ) -> Result<NativeValue, AnyError> {
   // Order of checking:
-  // 1. ArrayBufferView: Common and not supported by Fast API, optimise this case.
-  // 2. BigInt: Uncommon and not supported by Fast API, optimise this case as second.
-  // 3. Number: Common and supported by Fast API, optimise the common case third.
-  // 4. ArrayBuffer: Fairly common and not supported by Fast API.
+  // 1. BigInt: Uncommon and not supported by Fast API, optimise this case.
+  // 2. Number: Common and supported by Fast API.
+  // 3. Null: Very uncommon / can be represented by a 0.
+  let pointer = if let Ok(value) = v8::Local::<v8::BigInt>::try_from(arg) {
+    value.u64_value().0 as usize as *const u8
+  } else if let Ok(value) = v8::Local::<v8::Number>::try_from(arg) {
+    value.integer_value(scope).unwrap() as usize as *const u8
+  } else if arg.is_null() {
+    ptr::null()
```

Comment:
Why do the uncommon one first? Seems branch prediction would work better if you reverse this and check for number and null first.

Reply:
numbers enter the fast API call when optimized. This is the slow call so its more likely that the argument is a BigInt or null (least common)